code_generator/GdkCodeGenerator/Templates/UnityCommandPayloadGenerator.tt

```diff
@@ -18,14 +18,17 @@ namespace <#= qualifiedNamespace #>
 <# foreach (var commandDetails in commandDetailsList) {#>
         public class <#= commandDetails.CommandName #>
         {
+            /// <summary>
+            ///     Please do not use the default constructor. Use CreateRequest instead.
```

Comment:
Can you please explain why the def constructor should not be used?

Reply:
It forces a user to provide a reasonable value for each field

Reply:
We can't override the default constructor of a struct or make it private or internal

Reply:
It will be useful to put the explanation into the comment, if people don't know why they shouldn't do something they will not be discouraged

Reply:
Added comments :) PTAL

Reply:
nice! resolving